torch/_subclasses/functional_tensor.py

```diff
@@ -158,6 +161,15 @@ def __new__(cls, elem):
         )
         torch._C._set_throw_on_mutable_data_ptr(out)
         out.elem = elem
+
+        if torch.is_inference_mode_enabled():
+            if out.isRootTensor():
+                out._inference_mode_base = None
+                mode._storage_to_base[out.elem.storage()._cdata] = out
+            else:
+                out._inference_mode_base = mode._storage_to_base[
```

Comment:
is there cases where we wont see mode._storage_to_base does not exists? shall we be safe here and assign it to none if not found?

Reply:
I don't think so? We'd prefer an error